src/bios_handler.cpp

```diff
@@ -286,6 +286,43 @@ void IbmBiosHandler::saveAmmToVpd(const std::string& i_memoryMirrorMode)
     // TODO: Call write keyword API to update the value in VPD.
 }
 
+void IbmBiosHandler::saveAmmToBios(const std::string& i_ammVal)
+{
+    if (i_ammVal.size() != constants::VALUE_1)
+    {
+        std::cerr << "Bad size for AMM received, Skip writing to BIOS"
+                  << std::endl;
+        return;
+    }
+
+    std::string l_valtoUpdate =
+        (i_ammVal.at(0) == constants::VALUE_2) ? "Enabled" : "Disabled";
+
+    types::PendingBIOSAttrs l_pendingBiosAttribute;
+    l_pendingBiosAttribute.push_back(std::make_pair(
+        "hb_memory_mirror_mode",
+        std::make_tuple(
+            "xyz.openbmc_project.BIOSConfig.Manager.AttributeType.Enumeration",
+            l_valtoUpdate)));
+
+    try
+    {
+        dbusUtility::writeDbusProperty(
+            constants::biosConfigMgrService, constants::biosConfigMgrObjPath,
+            constants::biosConfigMgrInterface, "PendingAttributes",
+            l_pendingBiosAttribute);
+    }
+    catch (const std::exception& l_ex)
+    {
+        // TODO: Should we log informational PEL here as well?
+        logging::logMessage(
+            "DBus call to update lpar value in pending attribute failed. " +
```

Comment:
Log message should be "DBus call to update **AMM** value in pending attribute failed." ?

Reply:
Fixed